docs/getting-started/what-is-dotnet.md

```diff
@@ -73,19 +73,19 @@ CoreCLR, or the common language runtime found in .NET Core, is a lightweight run
 
 - A just-in-time (JIT) compiler. which compiles IL, or .NET intermediate language, to native machine code at runtime. On some architectures, the JIT compiler includes support for SIMD hardware acceleration.
 
-- An exception handling mechanism. The runtime, as well as library code, throws exceptions, which you can handle with try/catch statements.   
+- An exception handling mechanism, which allows you to handle exceptions with try/catch statements.   
 
 ##The Class Library. 
 
-The class library available in .NET Core is very much like the .NET Framework Class Library for the Windows desktop, with one major difference. Whereas on the Windows desktop is a monolithic library that is part of the operating system and is maintained by Windows Update, on .NET Core it is a modular and factored collection of individual libraries organized by functionality. 
+The class library available in .NET Core is very much like the .NET Framework Class Library for the Windows desktop, with one major difference. On the Windows desktop, it is a monolithic library that is part of the operating system and is maintained by Windows Update. On .NET Core it is a modular and factored collection of individual libraries organized by functionality. 
 
 One library, the Microsoft.NetCore.App, is included with the runtime and contains the basic types needed for all application development. Some of these include:
 
 -  Primitives, such as the Boolean type, signed and unsigned integral types, floating-point types, the Char structure.
   
 -  General-purpose exception classes, such as the ArgumentException, ArgumentNullException, and NullReferenceException types.
 
-- The task types, such as Task and Task<T>, to support asynchronous programming.
+- The task types, such as Task and Task&lt;T&gt;, to support asynchronous programming.
```

Comment:
Task types, such as Task and Task<T>.
Make them links and drop "to support async"